### ISSUE TITLE
multicast and broadcast mac Addresse are not catched.

### ISSUE THREAD

**Issue by janLo** [agdsn/sipa#3]
From the wikipedia:
> Ethernet frames with a value of 1 in the least-
> significant bit of the first octet[note 2] of the 
> destination address are treated as multicast 
> frames and are flooded to all points on the 
> network. While frames with ones in all bits of 
> the destination address (FF:FF:FF:FF:FF:FF) are 
> sometimes referred to as broadcasts, Ethernet 
> network equipment generally does not 
> distinguish between multicast and broadcast 
> frames. 

The user is not preventd to input addresses that mach the multicast vor broadcast criterion, AS the MacAddress validator only uses the regex `r'^(?:[0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2}$'` for validation.

**Comment by lukasjuhrich** [comment]
Ah, the commit must be in the master branch to close the issue, didn't know *that*…
@janLo When should this be done? I thought the develop branch should be merged into the master if you did major changes, which is not the case here. On the other hand, this is a more or less severe bug and should be applied immediately.

**Comment by lukasjuhrich** [comment]
@toothstone I suggest: Just because this issue is fixed in develop, it should *not* be closed.
As closing an issue is the final step, it implies that it does not have any relevance for the code any more. If the code fixing this is not yet in the master, this is simply not the case.

Because this project is now on Github, everybody has the possibility to clone the repository. Usually, the branch one is going to use is the `master`, as it is considered to be the most stable one.
Considering the master to be the point of view of an external observer, the latter would assume what I described above – *closed issues are nothing to be concerned about* – and only take a look at the open issues. In case of security bugs (which this one is), this is fatal.

The only argument to close this issue might be not to let it show up in the list of still open issues.
On the other hand, this can be achieved by the `fixed in develop` label you already set.
Just add `-label:"fixed in develop"` to the filter field in order to exclude these issues.

Therefore, I will reopen this issue and let Github close it *automatically* when merged into `master`.
If you have arguments I did not know about or see this differently, tell me so. I still am the one with less experience.
